### PR TITLE
14545 Upper case display for Corp Name Translation

### PR DESCRIPTION
**Pull request by lambert-alex**
Issue #: /https://github.com/bcgov/entity/issues/14545

Corresponding PR in Create UI https://github.com/bcgov/business-create-ui/pull/491

Description of changes:

corp name translation input is now capitalized.
add unit test for name translation input being capitalized.
align AddNameTranslation.vue component and the unit tests to the Create UI
App version 4.4.2 -> 4.4.3

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue [hunk 1]
```diff
@@ -5,27 +5,28 @@
       attach="#add-name-translation"
     />
     <!-- Name Translation form -->
-    <v-form v-model="nameTranslationForm" ref="nameTranslationForm" class="name-translation-form">
```
Comment (lambert-alex) [inline src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue:8]: This class was not used anywhere.

File: src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue [hunk 2]
```diff
@@ -5,27 +5,28 @@
...
+    <v-form v-model="nameTranslationForm" ref="nameTranslationForm">
       <v-text-field
         filled
         persistent-hint
         id="name-translation-input"
         label="Name Translation"
         v-model="nameTranslation"
         :rules="nameTranslationRules"
+        @input="nameTranslation = nameTranslation.toUpperCase()"
```
Comment (lambert-alex) [inline src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue:16]: The fix required by the ticket, all inputs are capitalized

File: src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue [hunk 3]
```diff
@@ -5,27 +5,28 @@
...
       />
 
       <div class="action-btns">
         <v-btn v-if="!isAddingTranslation"
           large outlined color="error"
-          id="name-translation-remove"
+          id="name-translation-btn-remove"
```
Comment (lambert-alex) [inline src/components/common/YourCompany/NameTranslations/AddNameTranslation.vue:22]: Aligning all of the btn IDs to be the same as the AddNameTranslation.vue in the Create UI

File: tests/unit/AddNameTranslation.spec.ts [hunk 4]
```diff
@@ -46,24 +47,26 @@ describe('Add Name Translation component', () => {
...
     expect(wrapper.find(addTranslationInput).exists()).toBeTruthy()
 
     // Verify Action btns and their default states
-    expect(wrapper.find(okBtn).exists()).toBeTruthy()
+    expect(wrapper.find(doneBtn).exists()).toBeTruthy()
     // done button not disabled anymore, validation done by validation function
-    expect(wrapper.find(okBtn).attributes('disabled')).toBeUndefined()
+    expect(wrapper.find(doneBtn).attributes('disabled')).toBeUndefined()
+
+    expect(wrapper.find(removeBtn).exists()).toBeFalsy()
```
Comment (lambert-alex) [inline tests/unit/AddNameTranslation.spec.ts:61]: Aligning the unit tests to the AddNameTranslation.vue in the Create UI

File: tests/unit/AddNameTranslation.spec.ts [hunk 5]
```diff
@@ -227,4 +239,12 @@ describe('Add Name Translation component', () => {
 
     wrapper.destroy()
   })
+
+  it('Input text is capitalized', async () => {
+    const wrapper = await wrapperFactory()
+    const translationInput = wrapper.find(addTranslationInput)
+    await translationInput.setValue('Lower case will be capitalized')
+
+    expect(wrapper.find(addTranslationInput).element.value).toBe('LOWER CASE WILL BE CAPITALIZED')
```
Comment (lambert-alex) [inline tests/unit/AddNameTranslation.spec.ts:248]: Unit test for the change requested in the ticket.

**Comment by lambert-alex** [comment]
The inputs in the Edit UI I also confirmed that they were being saved as upper case
Edit UI addNameTranslation

**Comment by severinbeauvais** [comment]
I believe there is some "uppercase" CSS for the list of name translations that you can now delete.

**Comment by severinbeauvais** [comment]
I see different disabled styling for the list of name translations above vs Create UI. It is easy to simply disable the actions (instead of the entire component) as in Create UI?

**Comment by severinbeauvais** [comment]
I see different styling for the Remove button, too. Check with Ethan or Yui on which is preferable and please update accordingly. Thanks :)

**Review by ChrisGabel23 — APPROVED** [review]

**Comment by lambert-alex** [comment]
> I see different disabled styling for the list of name translations above vs Create UI. It is easy to simply disable the actions (instead of the entire component) as in Create UI?

The problem is there are other different details, the remove button disappears when disabled in the edit ui vs just graying out in the create ui. The card is slightly wider in the Create UI I created a tech debt ticket [14752](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/14752) to address this.

**Comment by severinbeauvais** [comment]
> > I see different disabled styling for the list of name translations above vs Create UI. It is easy to simply disable the actions (instead of the entire component) as in Create UI?
> 
> The problem is there are other different details, the remove button disappears when disabled in the edit ui vs just graying out in the create ui. The card is slightly wider in the Create UI I created a tech debt ticket [14752](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/14752) to address this.

I'm OK with this. Thanks!

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**